lib/msf/core/handler/bind_aws_instance_connect.rb

```diff
@@ -142,11 +141,9 @@ def start_handler
         rescue Aws::EC2InstanceConnect::Errors::SerialConsoleSessionLimitExceededException => e
           print_error("Too many active serial console sessions. It takes 30 seconds to tear down a session after you've disconnected from the serial console in order to allow a new session.")
         rescue Aws::Errors::ServiceError => e
-          print_error(e.message) unless e.class == last_error_class
-          last_error_class = e.class
+          print_error(e.message)
```

Comment:
```
          vprint_error(e.message)
```

I think we were good to swap to vprint_error to align with bind_tcp's logic, and allow folk to ignore the message entirely if they wanted to